.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+### Are you looking for help?
```

Comment:
Issue template looks good. I think we should add some others question too. like `summary` of the issue that would be first question and other questions, order looks fine. 

if you want to see an example checkout this https://github.com/therajanmaurya/android-client-2.0/blob/development/.github/ISSUE_TEMPLATE.md